x/finality/gov.go

```diff
@@ -0,0 +1,29 @@
+package finality
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypesv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+
+	"github.com/babylonlabs-io/babylon/x/finality/keeper"
+	"github.com/babylonlabs-io/babylon/x/finality/types"
+)
+
+// NewResumeFinalityProposalHandler is a handler for governance proposal on resume finality.
+func NewResumeFinalityProposalHandler(k keeper.Keeper) govtypesv1.Handler {
```

Comment:
hmm why usage of the old way of handling proposals ? (which I think is discouraged) 

Wouldn't it be better to have separate message on finality provider message server which can be only executed by governance module ? (similar to parameters updates)